src/groupeddataframe/grouping.jl

```diff
@@ -1125,3 +1139,20 @@ function DataFrame(gd::GroupedDataFrame)
     resize!(idx, doff - 1)
     parent(gd)[idx, :]
 end
+
+"""
+    groupindices(gd::GroupedDataFrame)
+
+Return a vector of group indices for each row of `parent(gd)`.
+If for some row this function returns an index `i` this means that
```

Comment:
I'd phrase it this way: "Rows appearing in group `gd[i]` are attributed index `i`. Rows not present in any group are attributed `missing` (this can happen if `skipmissing=true` was passed when creating `gd`, or if `gd` is a subset from a larger `GroupedDataFrame`).

Reply:
A nice writeup. Fixed.